### PR TITLE
[C] Remove redis-rails

### LINKED ISSUE #2876: Remove redis-rails gem
See https://github.com/redis-store/redis-rails#a-quick-note-about-rails-52
See https://github.com/rails/rails/pull/31134

### PR DESCRIPTION
**Pull request by scryptmouse**
Resolves #2876

### CODE REVIEW

**Comment by zdavis** [comment]
@scryptmouse Linting issues on this one...

File: api/config/environments/development.rb [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
-    config.cache_store = :memory_store
-    config.public_file_server.headers = {
-      "Cache-Control" => "public, max-age=#{2.days.to_i}"
-    }
-  else
-    config.action_controller.perform_caching = false
-
-    config.cache_store = :null_store
-  end
+  #if Rails.root.join("tmp", "caching-dev.txt").exist?
```
Comment (zdavis) [inline api/config/environments/development.rb:17]: Should we just remove these lines instead of commenting?

Comment (scryptmouse) [reply api/config/environments/development.rb:17]: Yes, I meant to remove those actually, unless you don't want to have caching in development. I think it's beneficial, though.

**Comment by zdavis** [comment]
Cherry-picked this into zd/sass and will accept it in that PR.

**Pull request closed without merging**